dev-docs/new-distro.md

```diff
@@ -26,7 +26,7 @@ for the new distro. You can duplicate the most recent distro as a starting
 point. Then iterate on your change to `compile.go` by running:
 
 ```shell
-go run dockerfiles/compile.go && docker build --target $DISTRO_SHORT -t scratch .
+DOCKER_BUILDKIT=1 go run dockerfiles/compile.go && docker build --target $DISTRO_SHORT -t scratch .
```

Comment:
wait, I just noticed that DOCKER_BUILDKIT=1 should be attached to the docker build part of the command, not the go run part.

Reply:
😛 done